binary_search/binary_search.java

```diff
@@ -0,0 +1,46 @@
+/**
+ *
+ * @author jsroyal
+ */
+public class BinarySearch {
+
+    public static void main(String[] args) {
+        int arr[] = {1, 5, 35, 112, 258, 324},
+                search_arr[] = {1, 35, 112, 324, 67},
+                pos, i;
+        BinarySearch obj = new BinarySearch();
```

Comment:
>Use static method binarySearch instead. Refer to sample.

Using the static method in java loose the concept of OOP.
So my argument is can we update our sample code or more discuss it.